services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/switches/DefaultRulesSpec.groovy

```diff
@@ -129,6 +129,80 @@ class DefaultRulesSpec extends HealthCheckSpecification {
         }
     }
 
+    @Unroll
+    @Tags([TOPOLOGY_DEPENDENT, SMOKE_SWITCHES])
+    def "Able to install default multitable rule on an #sw.ofVersion \
+switch(#sw.dpId, install-action=#data.installRulesAction)"(Map data, Switch sw) {
+        setup: "Enabled multitable switch property on a switch(if it is not enabled yet)"
+        def initSwitchProperty = northbound.getSwitchProperties(sw.dpId)
+        def multitableIsEnabled = initSwitchProperty.multiTable
+        multitableIsEnabled ?: northbound.updateSwitchProperties(sw.dpId, northbound.getSwitchProperties(sw.dpId).tap {
```

Comment:
1. remove `multitableIsEnabled` variable. there is no reason to add it here
2. Can we stop using elvis `?:`  operator in such cases? It is hardly readable because it reverses the condition on more time. If you want it to be a one-liner use a short-circuit instead:
```
isMultitable && makeItSingleTable()
```
https://groovy-lang.org/operators.html#_short_circuiting